test/Database/Sql/Hive/Parser/Test.hs

```diff
@@ -1513,7 +1515,6 @@ testParser = test
         , "SELECT .01;"
         , "SELECT 1.0::INTEGER;"
         , "SELECT 1 AS 'foo';"
-        , "SELECT 1 || 2;"
```

Comment:
Oh Hive. Why document a feature if [it's not going to be implemented](https://issues.apache.org/jira/browse/HIVE-14580?focusedCommentId=15601822&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-15601822)?

Thanks for catching that!